python/src/deltachat/account.py

```diff
@@ -617,18 +617,17 @@ def set_location(self, latitude: float = 0.0, longitude: float = 0.0, accuracy:
     # meta API for start/stop and event based processing
     #
 
-    def run_account(self, addr=None, password=None, account_plugins=None, show_ffi=False):
-        from .events import FFIEventLogger
-
+    def run_account(self, addr=None, password=None, account_plugins=None, show_ffi=False, displayname=None):
         """get the account running, configure it if necessary. add plugins if provided.
 
         :param addr: the email address of the account
         :param password: the password of the account
         :param account_plugins: a list of plugins to add
         :param show_ffi: show low level ffi events
         """
+        from .events import FFIEventLogger
+
```

Comment:
I think the method is not showing up in the docs right now, because the import statement used to be above the docstring. Hopefully this gets fixed by this PR...